libraries/libopaevfio/opaevfio.c

```diff
@@ -1123,6 +1144,20 @@ STATIC int opae_vfio_init(struct opae_vfio *v,
 
 	mem_alloc_init(&v->iova_alloc);
 
+	result = opae_hash_map_init(&v->cont_buffers,
+				    19441, // num_buckets
```

Comment:
#define   OPAE_HASH_MAPS_SIZE 19441, // num_buckets

Reply:
Not all hash maps will be the same size.  This is implementation-specific.